lib/errors.js

```diff
@@ -72,6 +72,20 @@ Errors.prototype = {
     },
 
     /**
+     * Prepare error message.
+     *
+     * @param {Object} errorInfo
+     * @private
+     */
+    _prepareMessage: function(errorInfo) {
+        if (this._verbose && this._currentRule) {
```

Comment:
Just thinking about what Errors object should do. Reporters can format this kind of errors more wisely.
Nvm, it's another story.